table/partitioned_filter_block_test.cc

```diff
@@ -66,12 +66,13 @@ class PartitionedFilterBlockTest : public testing::Test {
   }
 
   uint64_t MaxFilterSize() {
+    uint32_t dont_care1, dont_care2;
```

Comment:
Can you explain this?

By the way, I think line 71 is better to be static_cast<> if we are sure about the type. If we are not sure about the type, calling in line 74 is wrong because it can be null pointer.